proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
+
+== MPWG
+Pros
+
+* Flatter governing structure as steering committee, marketing committee and specification committees share the same membership.
+* Addresses IP gap in existing MP development
+* Provides complete autonomy of how MP evolves as governing structure is not part of a non-MP specific working group
+* Provides a lower barrier to entry in terms of both cost and process for the MP technology
+* Continues a model that has produced multiple independent implementations of novel technologies over a number of years
+* Allows for large companies to sponsor specification development at a zero cost entry point
+* Less restrictive use of compatibility and logo usage
+
+Cons
+
+* Duplicate working group participation agreements need to be signed
+* My have duplicate fees depending on the membership level if both MPWG and CN4J/Jakarta are joined
+* Tends to not commit to longer term backwards compatibility 
+
+== CN4J
+Pros
+
+* Allows for parties interested in both MPWG and CN4J/Jakarta to sign one working group participation agreement
+* Addresses IP gap in existing MP development
+* Tends to focus on stability and backwards compatibility 
+* Has restrictions on how compatibility and branding are claimed (both Pro/Con)
+
+Cons
+
+* Currently has a higher entry cost and overall overhead to run
+* Currently has no zero entry participation model for larger companies to sponsor specification development
+* Introduces a conflict of interest between evolution and support of enterprise customers and their previous investment in Java EE with a shared steering committee
```

Comment:
In this proposal, since Jakarta and MicroProfile live under CN4J, from my understanding, they can use different process. I thought implementations can still just claim MicroProfile instead of Jakarta EE, though. Right? If true, why there will be a conflict?

Reply:
I think Scott is getting at is a single governing body (working group) that covers both Jakarta EE and MicroProfile *may* have a conflict of interest between their old investment in Java EE (and that customer base) and new, innovative technologies coming through the MP efforts.

Reply:
If there is a conflict, surely it should be resolved by the single steering committee. Otherwise, what signal are we going to send to the end users?

Reply:
Correct. A main point of conflict between the proposals is how the organizational elements above the specification committee operate in the context of  specifications and processes that operate differently. This is especially true if Jakarta specs start to consume MP specs without an intervening transition of the spec into a Jakarta spec process for example.